llmfoundry/models/layers/attention.py

```diff
@@ -16,6 +17,8 @@
 from llmfoundry.models.layers.fc import FC_CLASS_REGISTRY
 from llmfoundry.models.layers.norm import NORM_CLASS_REGISTRY
 
+log = logging.getLogger(__name__)
+
```

Comment:
```suggestion
```

If you're getting rid of all prints from this file, you're not logging anything / don't need it (also don't need to import it above.)